install.sh

```diff
@@ -561,6 +565,9 @@ do_install_tar() {
     install_airgap_tarball
     verify_tarball
     unpack_tarball
+
+    [ "${INSTALL_RKE2_SKIP_RELOAD}" = true ] && return
+
     systemctl daemon-reload
```

Comment:
I think this would better match the convention we use for other vars in this script
```suggestion
    if [ -z "${INSTALL_RKE2_SKIP_RELOAD}" ]; then
        systemctl daemon-reload
    fi
```

Reply:
Sounds good, updated.